src/Dev/Backtrace.php

```diff
@@ -149,11 +149,11 @@ public static function full_func_name($item, $showArgs = false, $argCharLimit =
         if ($showArgs && isset($item['args'])) {
             $args = [];
             foreach ($item['args'] as $arg) {
-                if (!is_object($arg) || method_exists($arg, '__toString')) {
+                if (is_object($arg)) {
+                    $args[] = get_class($arg);
+                } else {
```

Comment:
We shouldn't just assume that whatever string representation someone decided to output will be useful in backtraces. Even just the `DBField` implementation results in showing false information, since it looks like (for example) an int was passed directly when actually it was a `DBInt` object.

Reply:
This was originally done as part of the `__toString()` changes. I'd like to keep this even though it's not strictly _necessary_ anymore because giving bad debug info is just bad.